contents/handbook/growth/sales/new-sales.md

```diff
@@ -4,44 +4,68 @@ sidebar: Handbook
 
-Cameron reviews these and decides whether to route to [self-serve](/handbook/growth/sales/crm#inbound-self-serve-pipeline), schedule a demo, or pass to Simon (if they seem Very Large).  This handoff is currently a gut feel based on company size/revenue as Very Large customers may technically be considered low ICP per our [scoring](/handbook/growth/sales/icp). 
+- How closely do they match our [ICP](/handbook/who-we-are-building-for)?
+- Which products are they looking to use? What's their use case?
+- How large is their company? Revenue? Have they raised funding? (ie. will they pay >$20k for PostHog?)
+- What is the role of the person who filled out the form?
 
-**3. Initial demo**
+Most companies add friction here by making customers jump on a call first to qualify them - we don't do this because it's annoying to the customer. However, it's totally fine to ask them questions over email in advance of the demo to make sure you're making the best use of their time - just be specific. A few clarifying questions is fine, a 30 question survey is not. 
 
-Some general principles:
-- Be genuinely helpful - our customers usually already have specific problems in mind, so help them solve those first rather 
-- Get to the point - our ICP is technical and particularly sensitive to sales BS
-- Don't [promise](/handbook/growth/sales/overview#enterprise-customers) things we can't/don't want to deliver
-- Get into the actual demo as quickly as possible - you should spend most of your time here, not presenting PostHog's vision
+If you're pretty sure that they should be qualified out, you should still be helpful over email - some customers just use the form to get in touch and don't want to actually have a demo (e.g. they have a billing question or are asking about compliance things like HIPAA.)
 
-We have a simple deck (seriously, it's like 5 slides) - ask someone on the Sales team for an invite to our Pitch account so you can create customized versions. 
+### 3. Initial demo
 
-You generally want to make sure you cover:
+This is basically a combo of discovery and demo call - your objectives here are to:
 
-- Introduction, who is there, and what their roles are
-- Why they are here
+- Show the customer PostHog and get them excited
+- Find out as much as you reasonably can about them
+- Have a solid plan for next steps in place
+
+We have a simple deck (seriously, it's like 5 slides) - ask someone on the Sales team for an invite to our Pitch account so you can create customized versions. You should prepare a customized demo for each customer - the PostHog platform is super broad, so it's really easy to spend a bunch of time talking about something our ICP doesn't care about (which will really annoy them). 
```

Comment:
I'm not a fan of the term customized here - this implies we are going to do a different bespoke demo for every customer that comes in the door which is unfeasible and time-consuming.  For example, if a mobile native app customer comes in we don't have the ability to show specific events/session replays for them.  Instead we should be able to equate what we have in our production demo project to a mobile demo e.g. explain that Pageview events are like Screen events for mobile.

I think a better way to express this is 

> Come prepared to demo what the customer is looking for, don't throw the kitchen sink at them as it may be overwhelming.  Instead focus on the areas they have expressed interest in, and be prepared to hint at/show our other products if they sound interested in the demo.

Reply:
@leonposthog how would you articulate this? I was going off your feedback mainly that our demos feel too generic at the moment.

Reply:
I would lean more towards "A relevant and pointed demo" ; not necessarily customised or tailored specifically to them in terms of content, but more with regards to WHAT we show, and in what order we show it.

E.g "We desperately need to see how our customers our using our platform, and try to watch them use it"
- Go straight to Session Replay, then tie Replay into Analytics, then go from there.

Start with what their biggest problem/request is, stay there until they are happy, then move on to point two.

If you don't show what's important first, typically more senior people on the call will become distracted.

We don't want to fall into the trap of doing the same demo for each customer regardless of what they say at the beginning

Reply:
Agree with this approach but that's not a customized demo, it's understanding what the customer needs and showing it to them.

Reply:
Agreed - was just my response to Charles' edit, rather than disagreeing with your comment above

Reply:
Ok will update, I think I understand what you are both after here (ie. the same thing, just using different words)

> that's not a customized demo, it's understanding what the customer needs and showing it to them.

Hate to break it to you, but I think you just defined the word [customized](https://dictionary.cambridge.org/dictionary/english/customized)

Reply:
Updated, let me know if this new edit looks better?

Reply:
Yep - captured it well 👍

Reply:
I think this discussion feeds rather into the concept of being very familiar with the platform and _audible ready_ (to use some more frameworky sales terms). 

A personalised/tailored demo requires preparation and good understanding of the objectives and metrics a potential customer is looking for. I feel this can be qualifed within the first minutes of the meeting, which in return allows us to show relevant parts of the platform and comfirm with the prospect whether what they see matches their expectations. Creates a more involved, dynamic environment, too. 

A lot of text to agree with your agreement.